src/number.js

```diff
@@ -26,6 +26,33 @@ define([
 	numberFormatterFn, numberFormatProperties, numberNumberingSystem, numberParserFn,
 	numberParseProperties, numberPattern, numberSymbol, stringPad ) {
 
+function validateDigits( properties ) {
+	var minimumIntegerDigits = properties[ 2 ];
+	var minimumFractionDigits = properties[ 3 ];
+	var maximumFractionDigits = properties[ 4 ];
+
+	var minimumSignificantDigits = properties[ 5 ];
+	var maximumSignificantDigits = properties[ 6 ];
```

Comment:
We've been doing gradual migrations elsewhere. I see no point introducing new code with the old style just for consistency. Anything new shouldn't use `onevar` is easy enough to apply.